mk/travis.sh

```diff
@@ -73,6 +73,12 @@ if [[ ! "$TARGET_X" =~ "x86_64-" ]]; then
   fi
 fi
 
+# This stanza is separate from the above because MUSL requires a target install,
+# but does *not* require special linking / multilib.
+if [[ "$TARGET_X" =~ .*"-unknown-linux-musl" ]]; then
```

Comment:
Let's move the `rustup target add` from up above to here. We can change the condition to `[` "$TARGET_X" =~ .*"-unknown-linux-musl" || ! "$TARGET_X" =~ "x86_64-" ]]` or whatever Bashism.

That way, we have only one place where we do `rustup target add`. This will be important for other refactoring I am doing.